roles/openshift-applier/README.md

```diff
@@ -192,12 +192,15 @@ metadata:
 
 ### Filtering content based on tags
 
-The `openshift-applier` supports the use of tags in the inventory (see example above) to allow for filtering which content should be processed and not. The `filter_tags` variable/fact takes a comma separated list of tags that will be processed and only content with matching tags will be applied.
+The `openshift-applier` supports the use of tags in the inventory (see example above) to allow for filtering which content should be processed and not. The `filter_tags` variable/fact takes a comma separated list of tags that will be processed and only content with matching tags will be applied. Additionally, tags can be skipped in the same manner with the `exclude_tags` option. 
+
+**_NOTE:_** These two options are mutually exclusive.
```

Comment:
Don't think this is true anymore ... + probably should include some info re: not allowing the same tags as part of both options

Reply:
Also need to include some info on which one has precedence  - i.e.: if both `filter` (future `include`) and `exclude` are used and an entry has a list of tags whereas one exists in the `include` and another in `exclude` .